service/http.balancer/pom.xml

```diff
@@ -98,6 +98,24 @@
             <groupId>org.kaazing</groupId>
             <artifactId>k3po.junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
It would be better to move these as a dependency of the maven-plugin:
```xml
            <plugin>
                 <groupId>org.kaazing</groupId>
                 <artifactId>k3po-maven-plugin</artifactId>
</dependencies
    <dependency>
      <groupId>org.kaazing</groupId>
             <artifactId>specification.http</artifactId>
             <version>${k3po.version}</version>
          </dependency>
          <dependency>
              <groupId>org.kaazing</groupId>
              <artifactId>specification.httpx</artifactId>
              <version>${k3po.version}</version>
          </dependency>
          <dependency>
              <groupId>org.kaazing</groupId>
              <artifactId>specification.ws</artifactId>
              <version>${k3po.version}</version>
          </dependency>
             </plugin>
```